### ISSUE TITLE
Block theme: Applying "all" filter and searching for term resets to "curated".

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/pattern-directory#678]
**Describe the bug**
When I update my all/curated/community filter to "all" and search, it resets to "curated".

**To Reproduce**
1. Go to https://wordpress.org/patterns/?s=section&pattern-categories=gallery
3. Change filter from curated to "all"
4. Search for "section"
5. Notice that "all" is replaced with "curated".

**Expected behavior**
I would expect the filter option to not change.

**Screenshot**
![Screen Capture on 2024-04-18 at 09-59-39 (1)](https://github.com/WordPress/pattern-directory/assets/1657336/ccc4a761-c8bf-4a42-8cb4-3a13d766b756)

**Comment by ryelle** [comment]
I agree— I appreciate the filter logic here but I keep needing to toggle off "curated" to see a reasonable number of patterns with every page view.

**Comment by sinanisler** [comment]
yes, please

right now it is so annoying to browse patterns
